lib/osf-components/addon/components/search-result-card/component.ts

```diff
@@ -6,26 +6,40 @@ import { tracked } from '@glimmer/tracking';
 import Intl from 'ember-intl/services/intl';
 
 import SearchResultModel from 'ember-osf-web/models/search-result';
-import UserModel from 'ember-osf-web/models/user';
+import PreprintProviderModel from 'ember-osf-web/models/preprint-provider';
+
+
+const CardLabelTranslationKeys = {
+    project: 'osf-components.search-result-card.project',
+    project_component: 'osf-components.search-result-card.project_component',
+    registration: 'osf-components.search-result-card.registration',
+    registration_component: 'osf-components.search-result-card.registration_component',
+    preprint: 'osf-components.search-result-card.preprint',
+    file: 'osf-components.search-result-card.file',
+    user: 'osf-components.search-result-card.user',
+    unknown: 'osf-components.search-result-card.unknown',
+};
 
 interface Args {
     result: SearchResultModel;
+    provider?: PreprintProviderModel;
 }
 
 export default class SearchResultCard extends Component<Args> {
     @service intl!: Intl;
     @service store!: Store;
 
     @tracked isOpenSecondaryMetadata = false;
-    @tracked osfUser?: UserModel;
 
     @action
     toggleSecondaryMetadata() {
         this.isOpenSecondaryMetadata = !this.isOpenSecondaryMetadata;
     }
 
     get cardTypeLabel() {
-        return this.intl.t(`osf-components.search-result-card.${this.args.result.resourceType}`);
+        const preprintWord = this.args.provider?.preprintWord;
+        return (preprintWord && this.args.result.resourceType === 'preprint') ? preprintWord :
```

Comment:
i wondered if `preprintWord` is translatable -- [looks like](https://github.com/CenterForOpenScience/ember-osf-web/blob/1fd31777f2f1a4cbd0b17a7a0704e367cc41db86/app/models/preprint-provider.ts#L50) could do `provider.documentType.singular` (or `singularCapitalized`)